entity-types/browser-application/golden_metrics.yml

```diff
@@ -22,6 +22,14 @@ interactionToNextPaint75PercentileS:
     from: PageViewTiming
     eventId: entityGuid
     eventName: appName
+cumulativeLayoutShift75Percentile:
+  title: Cumulative layout shift (75 percentile)
+  unit: STRING
```

Comment:
I don't think STRING is the proper unit here.

Is there any specific reason why this should be an STRING?

I totally understand that any other values in the ENUM do not fit completely your needs, but I'd not put an STRING here. 

In many other places, we use just `COUNT`, which is the default.
The `COUNT` unit, is shown as no just a number in the UI.

Let me share some examples:

https://github.com/newrelic/entity-definitions/blob/1ce56fee1df3f75dacfac5c077e2a5711f3cb10d/entity-types/infra-container/summary_metrics.yml#L6

https://github.com/newrelic/entity-definitions/blob/1ce56fee1df3f75dacfac5c077e2a5711f3cb10d/entity-types/infra-awslambdafunctionalias/golden_metrics.yml#L46